project-base/easy-coding-standard.yml

```diff
@@ -3,6 +3,7 @@ imports:
 
 services:
     PhpCsFixer\Fixer\Strict\DeclareStrictTypesFixer: ~
+    SlevomatCodingStandard\Sniffs\Operators\DisallowEqualOperatorsSniff: ~
```

Comment:
```suggestion
    # @deprecated This will be moved from project-base to coding-standards package in next major version
    SlevomatCodingStandard\Sniffs\Operators\DisallowEqualOperatorsSniff: ~
```

Reply:
Add this same to framework package